docs/aspnet4x-mvc.md

```diff
@@ -189,23 +189,23 @@ public ActionResult Login()
 |`prompt` | Indicate the pipeline the intent of the request, such as, support enrollment of a new factor. | No |
 |`enroll_amr_values` |  A space-delimited, case-sensitive string that represents a list of authenticator method references. | No |
 
+> Note: When `prompt` is equals to `enroll_authenticator` you have to also indicate the `redirect_uri`. The `redirect_uri` cannot be the same as the normal OIDC flow `/authorization_code/callback` since there's no code involved in this flow. Instead, you have to specify a callback URI (which has to be added to your application's allowed URLs in your Okta dashboard) in your application where, for example, you can process the response and redirect accordingly. Also, it's expected you to already have a session before triggering the authenticator enrollment flow.
```

Comment:
```suggestion
> Note: When `prompt` is equals to `enroll_authenticator` you have to indicate the URL that Okta should send callback to after the user app sends the enrollment request. The `redirect_uri` cannot be the same as the normal OIDC flow `/authorization_code/callback` since there's no code involved in this flow. Instead, you have to specify a callback URI (which has to be added to your application's allowed URLs in your Okta dashboard) in your application where, for example, you can process the response and redirect accordingly. Also, it's expected you already have a session before triggering the authenticator enrollment flow.
```